app/controllers/cloud_volume_controller.rb

```diff
@@ -239,8 +255,8 @@ def new
     assert_privileges("cloud_volume_new")
     @volume = CloudVolume.new
     @in_a_form = true
-    @cloud_tenant_choices = {}
-    CloudTenant.all.each { |tenant| @cloud_tenant_choices[tenant.name] = tenant.id }
+    @storage_manager_choices = {}
```

Comment:
Here we are returning just the first level choices - only after we choose the storage manager, we can get the rest. I am not actually sure how this works for OpenStack, because it will return all cloud tenants, regardless of the cloud manager (i.e. if I have several OpenStack managers, I'll get these cloud tenants mixed).